config/locales/en.yml

```diff
@@ -17,3 +17,9 @@ en:
     permission_priority: "Priority"
     permission_visible: "Visible"
     permission_boolean: "Boolean"
```

Comment:
Lets also add more text about the format of permission. We can copy from README.